### PR TITLE
Feature/api/1715/add log drains by db

### PR DESCRIPTION
**Pull request by curzolapierre**
WIP: don't merge / close this PR. Add and delete routes will be added.

Related Scalingo/api#1715
Related Scalingo/api#1724

### CODE REVIEW

File: _scalingo_api/log_drains.md [hunk 1]
```diff
@@ -39,6 +39,7 @@ You will find how to enable this service using the CLI on the
 | port                | string  | Port of logs management service                                  |
 | drain-region        | string  | Region used by logs management service to identify their servers |
 | url                 | string  | URL of self hosted ELK                                           |
+| addon_uuid          | string  | UUID of the addon you want to add it to                          |
```
Comment (EtienneM) [inline _scalingo_api/log_drains.md:42]:
```suggestion
| addon_uuid          | string  | UUID of the addon you want to add the log drain to                          |
```

File: _scalingo_api/log_drains.md [hunk 2]
```diff
@@ -39,6 +39,7 @@ You will find how to enable this service using the CLI on the
 | port                | string  | Port of logs management service                                  |
 | drain-region        | string  | Region used by logs management service to identify their servers |
 | url                 | string  | URL of self hosted ELK                                           |
+| addon_uuid          | string  | UUID of the addon you want to add the log drain to                          |
```
Comment (EtienneM) [inline _scalingo_api/log_drains.md:42]: Your table does not look nice, you have the `|` too far away

File: _scalingo_api/log_drains.md [hunk 3]
```diff
@@ -151,6 +186,74 @@ Returns 201 Created
 }
 ```
 
+## Add a new Log Drain to an Addon
```
Comment (EtienneM) [inline _scalingo_api/log_drains.md:189]:
```suggestion
## Add a New Log Drain to an Addon
```

File: _scalingo_api/log_drains.md [hunk 4]
```diff
@@ -151,6 +186,74 @@ Returns 201 Created
...
+Example requests
+
+```shell
+curl -H 'Accept: application/json' -H 'Content-Type: application/json' -u ":$AUTH_TOKEN" \
+  -X POST https://$SCALINGO_API_URL/v1/apps/example-app/addons/ad-9be0fc04-bee6-4981-a403-a9dd4981bd1f/log_drains -d \
+  '{
+    "drain": {
+        "type": "ovh-graylog",
+        "token": "5af97be7-34e5-47b6-a016-8d0001228ffb",
+        "host": "tag1.logs.ovh.com"
```
Comment (EtienneM) [inline _scalingo_api/log_drains.md:217]: Shouldn't you add a port in this example? With the value `6514`?

Comment (curzolapierre) [reply _scalingo_api/log_drains.md:217]: The default port is 6514 and it is not editable so it not necessary to ask for it

File: _scalingo_api/log_drains.md [hunk 5]
```diff
@@ -282,4 +282,31 @@ curl -H 'Accept: application/json' -H 'Content-Type: application/json' -u ":$AUT
 Returns 204 No Content
 
 
+--- row ---
+
+## Delete a Log Drain from an Addon
```
Comment (EtienneM) [inline _scalingo_api/log_drains.md:287]:
```suggestion
## Delete a Log Drain From an Addon
```

File: _scalingo_api/log_drains.md [hunk 6]
```diff
@@ -282,4 +282,31 @@ curl -H 'Accept: application/json' -H 'Content-Type: application/json' -u ":$AUT
...
+
+--- row ---
+
+`DELETE https://$SCALINGO_API_URL/v1/apps/:app/addons/:addon_uuid/log_drains`
+
+### Parameter
+
+* `url`: logs drain URL to delete. You can get it by listing the logs drain of
```
Comment (EtienneM) [inline _scalingo_api/log_drains.md:295]:
```suggestion
* `url`: log drain URL to delete. You can get it by listing the log drains of
```

**Pull request merged**